addons/framework/statsTracking/fn_statsInitLocal.sqf

```diff
@@ -11,15 +11,17 @@
 
 waitUntil {not isNull player};
 
+Shots_Fired = 0;
```

Comment:
Use camelcase for variables and stop using underscores. An underscore should only begin a variable in sqf:

`_shotsFired`